src/lib/DateInput.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+  import { v4 as uuidv4 } from "uuid";
+  import { DefaultButton, ErrorMessage, FormElement } from "./index.js";
+
+  export let label: string;
+  export let errorMessage = "";
+  export let setToTodayButton = false;
+  export let value: string;
+
+  let id = uuidv4();
+  let dayId = `${id}-day`;
+  let monthId = `${id}-month`;
+  let yearId = `${id}-year`;
+
+  let dayString = "";
+  let monthString = "";
+  let yearString = "";
+  $: {
+    errorMessage = "";
+    const dayInCorrectRange =

```

Comment:
![image](https://github.com/user-attachments/assets/feee384b-47e6-4ad8-8822-877abc76cf38)
The date validation doesn't seem to work